bin/fk.sh

```diff
@@ -112,6 +117,9 @@ fk()
     fk config --key=autoUpdateEnabled --value=true > /dev/null
     kit-check-update
   ;;
+  help)
+    kit-help
+  ;;
   *)
     frun $cmd $@
```

Comment:
Wait i'm confused ? `frun` function is not getting removed. 
#39  is updated that `frun.rb` file is removed. `fk.rb` is renamed to `kit-run`
And why would this cause issues ?

Reply:
That's what I meant. We have `frun.rb` which is not actually used. I got confused that frun was coming from it. 

Since removing the rb file is not related to this the help message (the focus of this PR), it would be nicer if we split the PR into two. 
1. Help msg 
2. Removing the frun.rb and maybe document that frun is not actually coming from the file